### PR TITLE
feat: Add August newsletter.

### PR DESCRIPTION
**Pull request by lukesneeringer**
Also fix links in the March to June ones.
(The cleanup the generator does only applies to actual AIPs.)

### CODE REVIEW

File: pages/news/2020-08.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+In addition to the new AIPs under review, we have added the following guidance
+to existing AIPs:
+
+- [AIP-135](../135): Add "get" guidance for soft delete.
+  ([#526](https://github.com/aip-dev/aip.dev/pull/526))
+- [AIP-140](../140): Word segments in field names must not begin with a number.
+  ([#528](https://github.com/aip-dev/aip.dev/pull/528))
+- [AIP-151](../151): Add guidance for parallel operations.
+  ([#535](https://github.com/aip-dev/aip.dev/pull/535))
+- [AIP-158](../158): Clarify that `page_size` is always an opitonal field.
```
Comment (odsod) [inline pages/news/2020-08.md:54]: optional*

**Comment by jskeet** [comment]
Will try to get to this next week.

File: pages/news/2020-08.md (hunk 1 above)
Comment (lukesneeringer) [reply pages/news/2020-08.md:54]:
```suggestion
- [AIP-158](../158): Clarify that `page_size` is always an optional field.
```

File: pages/news/2020-08.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## AIPs under review
+
+### AIP-147: Sensitive fields
+
+> Sometimes APIs need to collect sensitive information such as private
+> encryption keys meant to be _stored_ by the underlying service but not
+> intended to be _read_ after writing due to the sensitive nature of the data.
+
+[Sensitive fields][aip-147] are fields that are stored by a service but, once
+set, not retrievable by the user, such as passwords or private keys. These are
```
Comment (jskeet) [inline pages/news/2020-08.md:23]:
```suggestion
[Sensitive fields][aip-147] are fields that are stored by a service but are not
retrievable by the user once they have been set. Examples of sensitive fields include
private keys and passwords. These are
```

(The number of commas in the original sentence caused me to take a long time to parse it.)

File: pages/news/2020-08.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+not particularly common in APIs, and often it is sufficient to set them as
+`INPUT_ONLY` (as described in AIP-203).
+
+However, there is a challenge in situations where the service needs to provide
+some kind of indication that a value has been provided, or provide an
+obfuscated value that the original user ought to recognize. This AIP provides a
+standard for each of those situations.
+
+A special thanks to [Michael Bleigh][] for writing this AIP.
```
Comment (jskeet) [inline pages/news/2020-08.md:32]: Possibly just "Special thanks" or "A special thank you"? It feels odd to use a singular article for something that is then at least apparently plural.

File: pages/news/2020-08.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+**Summary:** [AIP-147][] provides patterns for indicators around sensitive
+fields.
+
+We are aiming to approve AIP-147 on August 28. If you have feedback, please
+[leave a comment](https://github.com/aip-dev/aip.dev/pull/XYZ).
```
Comment (jskeet) [inline pages/news/2020-08.md:38]: Do we have a PR ready to link to? (I can't remember what order you do thigngs in.)

File: pages/news/2020-08.md (hunk 3 above)
Comment (lukesneeringer) [reply pages/news/2020-08.md:33]:
```suggestion
Special thanks to [Michael Bleigh][] for writing this AIP.
```

File: pages/news/2020-08.md (hunk 4 above)
Comment (lukesneeringer) [reply pages/news/2020-08.md:39]: I traditionally do this post-approval.

**Comment by lukesneeringer** [comment]
Doing some repo maintenance; I will reopen this as a new PR elsewhere.

**Pull request closed without merging**